common/src/main/java/tech/pegasys/web3signer/common/Web3SignerMetricCategory.java

```diff
@@ -24,7 +24,8 @@ public enum Web3SignerMetricCategory implements MetricCategory {
   ETH2_SLASHING_PROTECTION("eth2_slashingprotection"),
   FILECOIN("filecoin"),
   HTTP("http"),
-  SIGNING("signing");
+  SIGNING("signing"),
+  WEB3SIGNER("web3signer");
```

Comment:
Could use StandardMetricCategory.PROCESS instead of creating a new one?

Reply:
Agree, although I've created a new one to meet the requirements of the issue which named the metric using the  "web3signer" prefix. Could easily adjust to use StandardMetricCategory.PROCESS instead.

Reply:
Not sure if there's a standard naming convention for this...if there's not then I go with whatever is going to be easier for people to find, which might well be web3signer_version :)